data/aistack/milvus_totest.yaml

```diff
@@ -0,0 +1,46 @@
+global:
+  imagePullSecrets:
+  - application-collection
+cluster:
+  enabled: True
+image:
+  all:
+    registry: registry.suse.de 
+    repository: suse/sle-15-sp6/update/products/ai/totest/images/containers/milvus 
+    tag: 2.4.6
+    pullPolicy: IfNotPresent
+standalone:
+  persistence:
+    persistentVolumeClaim:
+      storageClass: local-path
+etcd:
+  replicaCount: 1
+  persistence:
+    storageClassName: local-path
+minio:
+  mode: distributed
+  replicas: 4
+  rootUser: "admin"
+  rootPassword: "adminminio"
```

Comment:
I suggest to use something like `%ROOT_PASSWORD%` here and below and then modify the yaml after downloading it with `file_content_replace`, using an openQA variable to store the root password.

Reply:
Also please, let's use for those parametes the naming convention `_SECRET_XXX_...`